Source/Instrument Addon.lvproj

```diff
@@ -586,6 +586,7 @@
 				<Item Name="Waveform Subtype Enum__ogtk__jki_vi_tester.ctl" Type="VI" URL="/&lt;vilib&gt;/addons/_JKI Toolkits/VI Tester/_support/Support.llb/Waveform Subtype Enum__ogtk__jki_vi_tester.ctl"/>
 				<Item Name="whitespace.ctl" Type="VI" URL="/&lt;vilib&gt;/Utility/error.llb/whitespace.ctl"/>
 			</Item>
+			<Item Name="Channel Properties.ctl" Type="VI" URL="../APIs/Scripting API/TypeDefs/Channel Properties.ctl"/>

```

Comment:
I think this .ctl needs to be part of the Scripting lib, and should not appear under the .lvproj dependencies.